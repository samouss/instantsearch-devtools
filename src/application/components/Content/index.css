.Content  {
  flex: 1;
  overflow-y: scroll;
  height: 100vh;
}
