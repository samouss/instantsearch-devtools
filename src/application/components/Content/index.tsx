import React, { SFC } from 'react';
import './index.css';

const Content: SFC = ({ children }) => <div styleName="Content">{children}</div>;

export default Content;
