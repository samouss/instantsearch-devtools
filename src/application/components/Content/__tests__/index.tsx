import React from 'react';
import { shallow } from 'enzyme';
import Content from '../index';

describe('<Content />', () => {
  it('exepct to render', () => {
    const wrapper = shallow(<Content>This is the children.</Content>);

    expect(wrapper).toMatchSnapshot();
  });
});
