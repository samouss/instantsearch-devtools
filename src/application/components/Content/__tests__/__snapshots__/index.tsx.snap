// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Content /> exepct to render 1`] = `
<div
  styleName="Content"
>
  This is the children.
</div>
`;
